modules/ROOT/pages/policy-mule3-custom-policy-references.adoc

```diff
@@ -143,7 +143,7 @@ Downloading and retrieving client information takes place in the background and
 
 The policy definition file defines the high level properties of the policy, the configurable parameters, labels, and tips that are visible as a UI when implementing the policy.
 
-The following example shows the IP White List policy definition YAML file:
+The following example shows the IP AllowList policy definition YAML file:
```

Comment:
Could we align to either use AllowList or Allowlist?

Reply:
Yes, great catch! That was a mistake. The term is "Allowlist". Done! Thanks!